app/config/ApplicationConfig.scala

```diff
@@ -51,6 +51,8 @@ object ApplicationConfig extends ApplicationConfig with ServicesConfig {
   override lazy val callbackUrl = loadConfig("tamc.external-urls.callback-url")
   override lazy val ivNotAuthorisedUrl = loadConfig("tamc.external-urls.not-authorised-url")
 
+  def ggSignInUrl: String = "/gg/sign-in"
```

Comment:
Wont this need a continue URL for the user coming back into the service? Or does this page do it from the redirect header by default, I cant recall?

Reply:
looked in to it. it as a parameter that can be added, safest to assume its needed than not need. have addded :)